### DISCUSSION TITLE
Export problems with Audiogridder stable on Cubase 10.5

### DISCUSSION THREAD

**Discussion by rrtmz** [apohl79/audiogridder#727] [Help]
Hello, I have a problem when exporting a mix from Cubase and Audiogridder is used in the project (mainly or exclusively on the master buss). There is no error during playback and the audio is correctly played, even with some spare CPU in Cubase, but when I export the mixdown to a wav file the beginning of the exported wav is muted for the first second or so (no audio for the first second and then the audio begins, with the first second cutted). I have had a similar error in another project during a time change in cubase (bpm slowing down through an automation): In playback everything is perfect, but in the exported file there is a moment of cutted (muted) audio.
The version of Audiogridder I'm using is the 'stable' one (1.1.1). Cubase 10.5 pro, Windows 10, RME interface.

**Comment by Henri75** [comment]
I got the same proble on mac/cubase 11 with 1.2 beta3 and 4. Sometimes I event got glitches at the beginning of the wave.
Really annoying.
My workaround  for now : move the start point of each tracks later (for example 2.0.0.0) then it will be ok !

By the way, I noticed too that there is a latency issue with offline rendering : when I export tracks processed  with audio mixdown  with AG inserts, then reimport the same track they are not synced : there is a small difference in time between the original and the exported one.
Happens only when I us some AG insert on tracks.

**Comment by apohl79** [comment]
Maybe you can play with different buffer sizes and offline vs realtime bouncing?

Also, do you see any errors or warnings in the plugin log files (especially buffering related)?

**Reply by yoshimodular** [reply]
This an old threat, but I’m having this issue exporting in Monterey with Nuendo 12 running AG in the same machine. AG works great but it glitches at the beginning of every export.
